### DISCUSSION TITLE
Love the website design

### DISCUSSION THREAD

**Discussion by bpoole6** [marcus-crane/utf9k#306] [General]
Hey Just want to say I love the website design. I had googled "why isn't discord used in business" and your article popped up. [One possible death for Discord](https://utf9k.net/blog/one-possible-death-for-discord/#:~:text=The%20most%20obvious%20point%20is,aren't%20aimed%20at%20business.)

**Comment by marcus-crane** [comment]
@bpoole6 Oh hey, thanks for the kind feedback!

Sorry I didn't see your post until now. I must have missed a notification when it was posted (and haven't ever had any reason to check the Discussion tab) but it was a nice message to discover during the holidays :slightly_smiling_face:
